Wikipedia/Code/WMFImageGalleryGradientViews.swift

```diff
@@ -0,0 +1,28 @@
+@objcMembers class WMFImageGalleryTopGradientView: WMFUnconfiguredGradientView {
+    public static let sharedStretchableImage = {
+        return WMFImageGalleryTopGradientView.init(frame: CGRect(x: 0, y: 0, width: 25, height: 44)).wmf_stretchableSnapshotImageWithAlphaChannel()
```

Comment:
is the bar always `44` or is it `32` in landscape?

Reply:
that’s why I used a stretchable img

Reply:
assuming it's a gradient that fades vertically, it should only be stretched horizontally, otherwise it gets distorted

Reply:
hmm i didn't notice any distortion, but may be because i was on simulator and wasn't zoomed in to 1:1... but ya i recall gradient re-sampling producing banding...